### PR TITLE
Add Course(s) & Book(s)

### PR DESCRIPTION
**Pull request by arsandev**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s)

### Description 
Add courses, books, and book's category.

### Why is this valuable (or not)?
I complete list of course and book references for free.

### How do we know it's really free?
We can access full of courses and books without account. Just by link.

### For book lists, is it a book? For course lists, is it a course? etc.
Book Lists: Its a Web Article.
Course Lists: Video.

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-programming-books-id.md [hunk 1]
```diff
@@ -27,6 +28,11 @@
 * [Koding C++ Dengan Qt](https://leanpub.com/koding-cpp-qt) *(Membutuhkan akun Leanpub atau email yang valid)*
 
 
+### CodeIgniter
+
+* [Tutorial CodeIgniter 3 & 4](https://www.petanikode.com/tutorial/codeigniter/)
```
Comment (eshellman) [inline free-programming-books-id.md:33]: add "in progress" indicator (translated)

File: free-programming-books-id.md [hunk 2]
```diff
@@ -61,6 +67,7 @@
 * [Belajar TypeScript](https://leanpub.com/belajar-typescript) *(Membutuhkan akun Leanpub atau email yang valid)*
 * [Mengenal JavaScript](http://masputih.com/2013/01/ebook-gratis-mengenal-javascript)
 * [Otomatisasi dengan gulp.js](https://kristories.gitbooks.io/otomatisasi-dengan-gulp-js/content/)
+* [Tutorial Dasar Javascript untuk Pemula](https://www.petanikode.com/tutorial/javascript/)
```
Comment (eshellman) [inline free-programming-books-id.md:70]: also in-progress

Comment (arsandev) [reply free-programming-books-id.md]: added.

File: free-programming-books-id.md (hunk 1 above)
Comment (arsandev) [reply free-programming-books-id.md]: added.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**